packages/node_modules/@webex/webex-core/test/unit/spec/services/services.js

```diff
@@ -100,6 +100,22 @@ describe('webex-core', () => {
       });
     });
 
+    describe('#fetchLoginUserInfomation', () => {
```

Comment:
we should add a test that verifies that fetchLoginUserInfomation() resolves even if the API request failed